tools/make/kube.mk

```diff
@@ -170,16 +170,9 @@ e2e-prepare: prepare-ip-family ## Prepare the environment for running e2e tests
 
 .PHONY: run-e2e
 run-e2e: e2e-prepare ## Run e2e tests
-	@$(LOG_TARGET)
-ifeq ($(E2E_RUN_TEST),)
+	
```

Comment:
why change this?

Reply:
if you want to run single e2e test locally, try `E2E_RUN_TEST=RateLimitCIDRMatch make run-e2e`.

Reply:
if you want to run single e2e test locally, try `E2E_RUN_TEST=RateLimitCIDRMatch make run-e2e`.